geode-core/src/main/java/org/apache/geode/internal/monitoring/ThreadsMonitoringImplDummy.java

```diff
@@ -32,9 +32,17 @@ public boolean startMonitor(Mode mode) {
   @Override
   public void endMonitor() {}
 
+  private static class DummyAbstractExecutor extends AbstractExecutor {
```

Comment:
I think this new class is not meant to be subclassed so I recommend removing `Abstract` from the name.

Reply:
done